static/js/panels/InstanceFormGuided.tsx

```diff
@@ -102,20 +104,80 @@ const InstanceFormGuided: FC = () => {
                 value={formik.values.name}
                 error={formik.touched.name ? formik.errors.name : null}
-                stacked
-              />
-              <Select
-                id="image"
-                name="image"
-                label="Image"
-                options={getImageOptions()}
-                onBlur={formik.handleBlur}
-                onChange={formik.handleChange}
-                value={formik.values.image}
-                error={formik.touched.image ? formik.errors.image : null}
-                required
-                stacked
               />
+              {formik.values.image ? (
+                <>
+                  <Row>
+                    <Col size={8}>
+                      <Input
+                        id="baseImage"
+                        name="baseImage"
+                        label="Base Image"
+                        type="text"
+                        value={
+                          formik.values.image.os +
+                          " " +
+                          formik.values.image.release +
+                          " " +
+                          formik.values.image.aliases.split(",")[0]
+                        }
+                        disabled
+                        required
+                      />
+                    </Col>
+                    <Col size={4} className="u-align-self-end">
+                      <SelectImageBtn
+                        appearance="link"
+                        caption="Change image"
+                        onSelect={handleSelectImage}
+                      />
+                    </Col>
+                  </Row>
+                  <Input
+                    id="architecture"
+                    name="architecture"
+                    label="Architecture"
+                    type="text"
+                    value={formik.values.image.arch}
+                    disabled
+                  />
+                  <Select
+                    id="instanceType"
+                    label="Instance type"
+                    name="instanceType"
+                    onBlur={formik.handleBlur}
+                    onChange={formik.handleChange}
+                    options={[
+                      {
+                        label: "Container",
+                        value: "container",
+                      },
+                      {
+                        label: "Virtual Machine",
+                        value: "virtual-machine",
+                      },
+                    ]}
+                    value={formik.values.instanceType}
+                    disabled={
+                      isContainerOnlyImage(formik.values.image) ||
+                      isVmOnlyImage(formik.values.image)
+                    }
+                  />
+                </>
+              ) : (
+                <>
+                  <Col size={4}>
+                    <Label>* Base Image</Label>
```

Comment:
I don't really like the fact that, with this `Label` here, we're artificially mimicking the behaviour of the "required `Input` field", because that might change in the future - by moving the asterisk to the end of the label, for example.

Instead, would it be possible to put an `Input` here and hide the actual `input` element, so that only the label gets shown? I know, it's hacky, but it's more future-proof and consistent with the other case where "Base Image" is an `Input`.

Reply:
Label has a required field that we can use here.